build.sbt

```diff
@@ -149,6 +149,16 @@ lazy val firrtl = (project in file("."))
   )
   .settings(mimaSettings)
 
+lazy val bridge = (project in file("bridge"))
+  .dependsOn(firrtl)
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq("org.chipsalliance" %% "chisel" % "6.0.0",
+      "org.apache.commons" % "commons-lang3" % "3.12.0",
+      "org.apache.commons" % "commons-text" % "1.9"),
+    addCompilerPlugin("org.chipsalliance" % "chisel-plugin" % "6.0.0" cross CrossVersion.full),
```

Comment:
I don't really want this project to depend on `chisel6` which is why am hesitant to upstream this code.

Reply:
Is it ok if only the `bridges` project depends on `chisel6`?

Reply:
oh sorry, I wasn't paying attention.

Yes. It is ok to have a separate bridges project. I won't publish it to maven though. But I can help you publish it if you want to.

Reply:
Thanks. I need to do a bit more work with this before we merge. 

Maybe publish in the future, but it isn't necessary yet.

Reply:
OK. Please ping we when you want to merge.